ntoskrnl/ps/query.c

```diff
@@ -49,6 +49,189 @@ PsReferenceProcessFilePointer(IN PEPROCESS Process,
     return Section ? STATUS_SUCCESS : STATUS_UNSUCCESSFUL;
 }
 
+/**
```

Comment:
Not sure we need all this documentation for simple debug helpers 😅

I would suggest you put them inside an `#if DBG ... #endif` because they are only used in this case, and in release mode compilation they wouldn't be necessary (the corresponding DPRINT1's being nopped --- ok maybe verify this assertion of mine; I don't know whether our GitHub actions compile some artifacts in release mode; otherwise, you could try running your pr in our buildbots, there's the GCCLin_x86_Release).